_sass/custom/custom.scss

```diff
@@ -127,3 +127,21 @@ $link-color: #137752;
 //lg: $content-width + $nav-width,
 //xl: 1400px
 //);
+
+li a:link,
```

Comment:
I think custom CSS overrides (the naming of this file is rather confusing) should rather go into `_includes/head_custom.html` so we can be sure that the custom CSS loads _after_ the theme and overrides rules correctly (no `!important` needed).

Also, this might be a shorter way to do it:

```css
a:not([class]),
a:not([class]):hover,
.btn,
.btn:hover {
  background-image: none;
}
```